docs/upload_mapping_file.md

```diff
@@ -65,7 +65,7 @@ For any given error, you can access the file path, line number, and a code snipp
    }
    ```
 
-2. [Create a dedicated Datadog API key][2] and export it as an environment variable named `DD_API_KEY` (alternatively, pass it as a task property or if you have `datadog-ci.json` file in the root of your project, it can be taken from `apiKey` property there).
+2. [Create a dedicated Datadog API key][2] and export it as an environment variable named `DD_API_KEY` or `DATADOG_API_KEY` (alternatively, pass it as a task property or if you have `datadog-ci.json` file in the root of your project, it can be taken from `apiKey` property there).
```

Comment:
```suggestion
2. [Create a dedicated Datadog API key][2] and export it as an environment variable named `DD_API_KEY` or `DATADOG_API_KEY`. Alternatively, pass it as a task property, or if you have `datadog-ci.json` file in the root of your project, it can be taken from an `apiKey` property there.
```